tests/test_fx/test_codegen/test_activation_checkpoint_codegen.py

```diff
@@ -84,21 +114,31 @@ def test_act_ckpt_python_code_torch11():
 
     # check ops are annotated with ckpt
     ckpt_nodes = ['mlp1_linear1', 'mlp1_linear1_1', 'mlp2_linear1', 'mlp2_linear1_1']
+    offload_starts = ['mlp2_linear1']
     for node in graph.nodes:
         if node.name in ckpt_nodes:
             assert hasattr(node, 'activation_checkpoint')
 
-    # assert checkpoint function will be generated
+            # annotate the selected node for offload
+            if node.name in offload_starts:
+                setattr(node, 'activation_offload', True)
+
+    # assert checkpoint function will be generated and
+    # the offload option is correct
     code = graph.python_code('self').src
-    assert 'checkpoint_0' in code and 'checkpoint_1' in code
+    assert 'colossalai.utils.activation_checkpoint.checkpoint(checkpoint_0, False, x)' in code and \
+    'colossalai.utils.activation_checkpoint.checkpoint(checkpoint_1, True, x)' in code
 
     # recompile and verify the outputs are consistent
     gm = GraphModule(model, graph)
     gm.recompile()
     fx_out = gm(data)
     assert torch.equal(non_fx_out, fx_out)
 
+    gpc.destroy()
```

Comment:
I have added those gpc.destroy() functions to destroy the colossalai.launch process, hope it works